homeassistant/components/imap/strings.json

```diff
@@ -42,7 +42,8 @@
           "folder": "[%key:component::imap::config::step::user::data::folder%]",
           "search": "[%key:component::imap::config::step::user::data::search%]",
           "custom_event_data_template": "Template to create custom event data",
-          "max_message_size": "Max message size (2048 < size < 30000)"
+          "max_message_size": "Max message size (2048 < size < 30000)",
+          "enforce_polling": "Enforce polling even if the server supports Push-IMAP"
```

Comment:
```suggestion
          "force_polling": "Force polling even if the server supports Push-IMAP"
```

Reply:
Should we tell them in the description that if updates are unreliable, they can disable push updates by doing XXXX?

Reply:
If have implemented an `enable_push` option that is enabled by default and can be turned off. I added comment to indicate that IMAP-Push can be turned off if  updates are unreliable.